tests/foreman/api/test_provisioning.py

```diff
@@ -138,20 +138,32 @@ def test_rhel_pxe_provisioning(
     # Host is not blank anymore
     provisioning_host.blank = False
 
+    provisioning_host.key_filename = None
     # Wait for the host to be rebooted and SSH daemon to be started.
     provisioning_host.wait_for_connection()
 
+    # For RHEL9 since root login is disabled
+    disable_ssh_cmd = (
```

Comment:
This will be true for all your provisioning tests for RHEL9 so can we keep this common for all the  tests?

Reply:
@shubhamsg199 How about creating a new helper to run and verify this REX job to enable or permit root login in ProvisioningSetup host_helpers (run just for EL9) and run other REX task for RHEL7/8/9 to verify hostname as we do now?